core/lib/l1_contract_interface/Cargo.toml

```diff
@@ -13,6 +13,7 @@ readme = "README.md"
 [dependencies]
 zksync_types = { path = "../types" }
 zksync_prover_interface = { path = "../prover_interface" }
+zksync_config = { path = "../config" }
```

Comment:
Is it used anywhere?

Reply:
Nope, it was used in a previous version where `zksync_config::L1BatchCommitDataGeneratorMode` was being used. Removing it asap.